monitoring/README.md

```diff
@@ -92,47 +157,108 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
 a) API Server Monitoring: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-monitoring.png" width="60%" height="80%" title="apiserver-monitoring"> 
 
```

Comment:
FYI: the panels using PromQL's `histogram_quantile` function have spurious precision (because that function does), and you could help the reader by listing the histogram bucket boundaries in the panel's description.

Reply:
Could you open an issue for this feature improvement? I would like to address this into a new PR. Many thanks!

Reply:
Done. See #2466 .

Reply:
Thank you!